include/secrets.h

```diff
@@ -5,20 +5,25 @@
 #include <array>
 #include <string>
 #include "Machine.h"
+#include "conf.h"
 
 namespace secrets
 {
   namespace machine_data
   {
     const std::string machine_name = "EXAMPLE_MACHINE";
-    static constexpr Machine::MachineIDType machine_id = 0x12345678; /* Machine connected to the ESP32 */
-    static constexpr std::array<card::uid_t, 10> whitelist /* List of RFID tags whitelisted, regardless of connection */
+    static constexpr uint16_t machine_id = (uint16_t)12345678; /* Machine connected to the ESP32 */
```

Comment:
here you are casting explicitly, this is a big no-no, and the piece of code is exaclty why.
With your cast you are telling the compiler "trust me bro", but...
uint16_t is max 65.536, you are putting in 12.345.678. 
The compiler would be screaming at you if ti wasnt for that cast